.gitignore

```diff
@@ -1,5 +1,5 @@
+*.log
 coverage/
 dist/
 node_modules/
 package-lock.json
-yarn.lock
```

Comment:
The reason I added both `package-lock.json` and `yarn.lock` is that they are not respected by users of the package. It's only useful for applications. npm won't publish the lock file and yarn will ignore any lock files in packages.

Reply:
lock files are used to enable reproducible creation in ci

Reply:
Valid point, I'll allow it... :D